### PR TITLE
Stop using deprecated library in package_config

### PR DESCRIPTION
**Pull request by jathak**

### CODE REVIEW

**Comment by jathak** [comment]
Do you know why the Windows build here is failing? https://travis-ci.org/github/google/dart_cli_pkg/jobs/687250603

**Comment by nex3** [comment]
Ugh, it looks like the Dart SDK we're using is in a format that the Chocolatey version converter doesn't like. It looks like the SDK changed its dev version format (the current prerelease is `2.9.0-9.0.dev`, whereas older versions were `2.8.0-dev.17.0`. Chocolatey meanwhile seems like it doesn't allow the first component after a hyphen to be a number, so it's rejecting this new scheme as invalid.

It looks like the [Chocolatey Dart SDK versions](https://chocolatey.org/packages/dart-sdk#versionhistory) are now of the format `Dart SDK 2.9.0.9-c-000-dev`, where I guess the `c` exists just to work around the no-number restriction? We'll have to adjust our Dart SDK conversion to match.

**Comment by jathak** [comment]
@nex3: Does the conversion for the new Dart SDK format look good?

File: lib/src/chocolatey.dart [hunk 1]
```diff
@@ -69,8 +69,8 @@ String get _chocolateyVersion {
 /// dots in prerelease versions.
 ///
 /// The Dart SDK doesn't use the same logic for Chocolatifying pre-release
-/// versions that Sass does. Instead it transforms `A.B.C-dev.X.Y` into
-/// `A.B.C.X-dev-Y`.
+/// versions that Sass does. Instead it transforms `A.B.C-X.Y-dev` into
```
Comment (nex3) [inline lib/src/chocolatey.dart:72]:
```suggestion
/// versions that Sass does. Instead it transforms `A.B.C-X.Y.dev` into
```

File: lib/src/chocolatey.dart [hunk 2]
```diff
@@ -79,9 +79,18 @@ String get chocolateyDartVersion {
       "${dartVersion.major}.${dartVersion.minor}.${dartVersion.patch}");
 
   var prerelease = List.of(dartVersion.preRelease);
-  var firstInt = prerelease.indexWhere((value) => value is int);
-  if (firstInt != -1) result.write(".${prerelease.removeAt(firstInt)}");
-  result.write("-${prerelease.join('-')}");
+  if (prerelease.first is int) {
+    // Handle new style of version for Dart prereleases >=2.9
```
Comment (nex3) [inline lib/src/chocolatey.dart:83]: Probably a good idea to include an example of each format in each clause.

**Pull request merged**